draft-ietf-httpbis-message-signatures.md

```diff
@@ -275,74 +277,38 @@ The following table shows example canonicalized values for different content ide
 
 ## Specialty Content Fields {#specialty-content}
 
-Content not found in an HTTP header can be included in the signature base string by defining a content identifier and the canonicalization method for its content.
+Content not found in an HTTP header can be included in the signature base string by defining a content identifier and the canonicalization method for its content. 
 
 To differentiate specialty content identifiers from HTTP headers, specialty content identifiers MUST start with the "at" `@` character. This specification defines the following specialty content identifiers:
 
-@request-target
-: The target request endpoint. ({{content-request-target}})
-
 @signature-params
 : The signature metadata parameters for this signature. ({{signature-params}})
 
-Additional specialty content identifiers MAY be defined and registered in the HTTP Signatures Specialty Content Identifier Registry. ({{content-registry}})
+@method
+: The method used for a request. ({{content-request-method}})
 
-### Request Target {#content-request-target}
+@target-uri
+: The target URI for a request. ({{content-target-uri}})
```

Comment:
absolute?

Reply:
Yes, I believe that was the intent here, we can add that to the definition.